docs/source/concepts.rst

```diff
@@ -0,0 +1,33 @@
+.. _concepts:
+
+Here I preparing to firstly introduce the differences between task, rpc, and broadcast and their usage scenario, because the task in kiwipy is slightly different from the task defined in RabbitMQ's tutorial. The task in kiwipy is bidirectional and the return value can be obtained in client, which is more like a simple rpc. Thus easy to be confused with rpc mode.
```

Comment:
```suggestion
Here we introduce the meaning of task, rpc, and broadcast and their usage scenarios, since the meaning of a the task in ``kiwipy`` differs slightly from the one in RabbitMQ's tutorial.
```

I suggest to move the sentence "The task in kiwipy is bidirectional and the return value can be obtained by the client, which is more like a simple rpc. Thus easy to be confused with rpc mode." under the "Task" section